### PR TITLE
preprod testnet

### PR DESCRIPTION
**Pull request by zuzunker**

### CODE REVIEW

**Review by zuzunker — COMMENTED** [review]
/check

File: packages/yoroi-extension/app/api/ada/lib/storage/database/prepackaged/explorers.js [hunk 1]
```diff
@@ -134,6 +134,19 @@ const CardanoTestnetExplorers: Array<$ReadOnly<ExplorerRow>> = [
   },
 ];
 
+const CardanoPreprodTestnetExplorers: Array<$ReadOnly<ExplorerRow>> = [
+  {
+    ExplorerId: 4_00,
```
Comment (neuodev) [inline packages/yoroi-extension/app/api/ada/lib/storage/database/prepackaged/explorers.js]: The duplication of the `ExplorerId` to be [400](https://github.com/Emurgo/yoroi-frontend/blob/78761b871aee55fafbe6ab447813932b1b083e1f/packages/yoroi-extension/app/api/ada/lib/storage/database/prepackaged/explorers.js#L126) is causing the extension to crash when restoring a wallet on the `Legacy` testnet. The Id must be unique. It took me a while to debug this :)

**Review by neuodev — CHANGES_REQUESTED** [review]

File: packages/yoroi-extension/app/api/ada/lib/storage/database/prepackaged/explorers.js (hunk 1 above)
Comment (zuzunker) [reply packages/yoroi-extension/app/api/ada/lib/storage/database/prepackaged/explorers.js]: oh heck, good catch! Thank you, @AhmedIbrahim336, fixing it now!

Comment (zuzunker) [reply packages/yoroi-extension/app/api/ada/lib/storage/database/prepackaged/explorers.js]: Updated now, and replaced with cardanoscan, plz check again

Comment (neuodev) [reply packages/yoroi-extension/app/api/ada/lib/storage/database/prepackaged/explorers.js]: Sounds good, thanks @vsubhuman

**Review by neuodev — APPROVED** [review]

**Review by yushih — APPROVED** [review]

**Review by zuzunker — COMMENTED** [review]
/check

**Pull request merged**